lab-kyle/test/test-fs-function.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const read = require(__dirname + '/../lib/fs-function');
+const expect = require('chai').expect;
+
+describe('testing module lib/fs-function.js', function() {
+  describe('testing function bufferfiles()', function() {
+    it('should return the first 8 bits of a text file in hex ', function(done){
+      read.bufferfiles(__dirname + '/../data/one.txt', function(err, data){
+        expect(err).to.equal(err);
```

Comment:
This is essentially saying expect(true).to.equal(true);  I suggest you add a more meaningful test assertion, like since the data file exists expect(null).to.equal(err) or something like that :)